pkg/scheduler/util/backoff_utils_test.go

```diff
@@ -64,13 +64,17 @@ func TestBackoff(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		duration := backoff.GetEntry(test.podID).getBackoff(backoff.maxDuration)
-		if duration != test.expectedDuration {
-			t.Errorf("expected: %s, got %s for %s", test.expectedDuration.String(), duration.String(), test.podID)
-		}
-		clock.t = clock.t.Add(test.advanceClock)
-		backoff.Gc()
+		t.Run(test.expectedDuration.String(), func(t *testing.T) {
+			duration := backoff.GetEntry(test.podID).getBackoff(backoff.maxDuration)
+			if duration != test.expectedDuration {
+				t.Errorf("expected: %s, got %s for %s", test.expectedDuration.String(), duration.String(), test.podID)
+			}
+
+			clock.t = clock.t.Add(test.advanceClock)
```

Comment:
Hmmm... this is another test that expects the cases to be run in sequence and carries over state.